### ISSUE TITLE
Add link to Github project repo on the site

### ISSUE THREAD

**Issue by sara-02** [Diversifyingopensource/diversifyingopensource.github.io#10]
On the project site `https://diversifyingopensource.github.io/`, there is no section that directs us to the Github repo for this project.
We can either add a `fork me on github` logo, or in the contact us page add a section that provides the GitHub link.
This will increase the project visibility, inviting more people to pitch in their ideas, add issues, or in improving existing content. 
Otherwise one has to perform an explicit search to find the Github link of the project.(like I had to for filling this issue)

**Comment by nerdyvaishali** [comment]
Thanks for opening an issue, Sarah. I agree with adding a github link somewhere on a website. Would you like to send a PR for the same? Will be happy to review and merge it.

**Comment by sara-02** [comment]
@nerdyvaishali Sure I would love to. However it might take me some time to figure it out, HTML is not my area of expertise.

**Comment by iamneha** [comment]
@sara-02 let me know if I can raise PR for this. I would love to contribute.

**Comment by sara-02** [comment]
@iamneha Go ahead, you can help with this issue :)

**Comment by iamneha** [comment]
@sara-02 hey I added this and attached the screenshot also.
The design is not modular and you can see this only on home page. I think we should open more issue to make codebase cleaner and modular. So many unnecessary codes are there.

**Comment by sara-02** [comment]
@iamneha You can open other issues for code refactoring separately.
I guess this site was put to support bare minimal information with a focus on meetups.

**Comment by iamneha** [comment]
@sara-02 Yes even I was thinking the same but still, this site could be better and people show more interest if the content is presented properly on site. Because coming to GitHub everytime and search for the meetup in issue and go through all those comments is waste of time in future. If everything presented properly on site then it is very easy to get more people involved.
With the help of beginner contribution, this site could be better and they can grow this on the big level.
I will plan something on weekend and open more issue to make it better. HTML is not even my area of expertise but I'll try to solve the issue.
I think we should invite @Rakhisharma to make it better.

**Comment by Rakhisharma** [comment]
@iamneha @sara-02 this is an open source template we are using. When we started this meetup group I and Vaishali have already refactored it a lot. I understand there is still a lot to do. Please feel free to open issues, would love to look into it. And definitely, we will try to solve it. Feel free to send PR :)
